alexthompson207/whats-cookin
# Feature/user stories view all recipes

## Is this a fix or a feature?

This is a feature.

## What is the change?

As a user, I  am able to view a list of all recipes.
As a user, I am able to click on a recipe to view more information including directions, ingredients needed, and total cost.
As a user, I am able to filter recipes by multiple tags.
As a user, I am able to search recipes by their name or ingredients.

## What does it fix?

## Where should the reviewer start?

Start in the `scripts.js` file.

## How should this be tested?

Running `src/open index.html` once in project directory. User should be able to see a list of all recipes, filter by tag, search by recipe name or ingredient name. Should be able to view the instructions and ingredients of the recipe they clicked on, as well as the total cost for the recipe.

## Screenshots (if appropriate):
![screencapture-file-Users-alexthompson-turing-mod-2-projects-whats-cookin-src-index-html-2021-02-13-11_56_21](https://user-images.githubusercontent.com/71950194/107858679-8e30ad00-6df2-11eb-9175-cc367934d1bf.png)
![screencapture-file-Users-alexthompson-turing-mod-2-projects-whats-cookin-src-index-html-2021-02-13-11_55_57](https://user-images.githubusercontent.com/71950194/107858682-90930700-6df2-11eb-9757-95c21bdddcf8.png)
